install.ps1

```diff
@@ -0,0 +1,6 @@
+$url=(Invoke-WebRequest -Uri https://api.github.com/repos/meshcloud/collie-cli/releases/latest | Select-String -Pattern '.*\"browser_download_url\":\"(.*windows-msvc.exe).*' | ForEach-Object {($_.matches.groups[1]).Value})
+
+New-Item -ItemType directory -Path "${HOME}\collie-cli"
+Invoke-WebRequest -Uri $url -OutFile "${HOME}\collie-cli\collie.exe"
+
+Write-Host "Please add ${HOME}\collie-cli\collie.exe to your path environemnt variable."
```

Comment:
```suggestion
Write-Host "Please add ${HOME}\collie-cli\collie.exe to your path environment variable."
```

Reply:
For me to get `collie` as a command to work I had to:
* Remove `.exe` as it forces me to run `collie.exe tenant list` for example.
* Remove `collie.exe` from the path. Instead, I added `C:/Users/Jelle/collie-cli/` to the path. Other approaches did not work.